static/scss/pages/settings.scss

```diff
@@ -0,0 +1,116 @@
+#settings-main {
+    .c-settings-headline {
+        @include text-title-2xs;
+        padding: $spacing-lg $spacing-lg $spacing-md ;
+        color: $color-grey-60;
+    }
+
+    .c-meta--links {
+        display: none;
+    }
+
+    .c-settings-form--heading {
+        @include text-title-3xs;
+        color: $color-grey-50;
+        font-weight: normal;
+    }
+
+    .c-settings-form--footer {
+        background-color: transparent;
+        text-align: center;
+        display: block;
+    }
+
+    .c-settings-form--field {
+        border-bottom: 1px solid $color-grey-10;
+    }
+
+    .c-settings-form--inline-warning {
+        box-shadow: none;
+        text-align: start;
+        margin-left: 0;
+    }
+
+    .c-settings-form--submit {
+        padding: $spacing-sm $spacing-2xl;
+    }
+
+    .c-settings-form--notification {
+       box-shadow: var(--dropShadowLight); 
+       background-color: $color-white;
+       margin: $spacing-md;
+       padding: $spacing-md;
+       border-radius: $border-radius-md;
+    }
+
+    .c-settings-form--notification-heading {
+        @include text-title-3xs;
+        color: $color-grey-60;
+        border-left-width: 5px;
+        border-left-style: solid;
+        margin: 0;
+        padding: $spacing-sm $spacing-md;
+
+        &.t-error {
+            border-color: $color-error;
+        }
+        &.t-warning {
+            border-color: $color-warning;
+        }
+    }
+
+    .c-settings-form--notification-description {
+        border-left-width: 5px;
+        border-left-style: solid;
+        margin: 0;
+        padding: $spacing-sm $spacing-md;
+
+        &.t-error {
+            border-color: $color-error;
+        }
+        &.t-warning {
+            border-color: $color-warning;
+        }
+    }
+
+    @media screen and #{$mq-md} {
+        & {
+            display: flex;
+        }
+
+        .c-meta {
+            width: $content-sm;
+        }
+
+        .c-meta--links {
+            display: block;
+            padding: 0 $spacing-lg $spacing-md;
+
+            a {
+                @include text-body-lg;
+
+                img {
+                    margin-right: $spacing-sm;
+                    stroke: #5953F3;
+                    width: 15px;
```

Comment:
Don't know if there's an appropriate token for this somewhere?

Reply:
Not really. I'm fan of `1rem` so it'll get larger to match their text size. That might work?

Reply:
Oops, looks like my comment here got lost. So I went ahead and copied the exact text size: 87d83b7aa837ce6da223b9e6ccf67415daf0fac1